ivy/stateful/module.py

```diff
@@ -647,6 +650,8 @@ def __call__(
             track_submod_call_order,
             expected_submod_rets,
         )
+        # if not v:
```

Comment:
if we don't need these lines, then we don't need this comment.

Reply:
done